game-app/game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -152,9 +149,7 @@ public boolean hasPath(final String path) {
    *     resource. Do not use '\' or File.separator)
    */
   public @Nullable URL getResource(final String inputPath) {
-    return findResource(inputPath) //
-        .or(() -> findResource(inputPath))
-        .orElse(null);
```

Comment:
This is the regression I was talking about in the other PR, findResource was called twice